boards/seeed_xiao_esp32c6.json

```diff
@@ -0,0 +1,47 @@
+{
+  "build": {
+    "core": "esp32",
+    "extra_flags": [
+      "-DARDUINO_XIAO_ESP32C6",
+      "-DARDUINO_USB_CDC_ON_BOOT=1",
+      "-DARDUINO_USB_MODE=1"
```

Comment:
Are these extra flags still required taking into account that Arduino is not supported.